### ISSUE TITLE
Feat(RSS-Link): aria-label is missing

### ISSUE THREAD

**Issue by AugustinMauroy** [nodejs/nodejs.org#6414]
## Problem

Here the Lighthouse report:

<img width="815" alt="Capture d’écran 2024-03-03 à 12 08 20" src="https://github.com/nodejs/nodejs.org/assets/97875033/876b2eef-04fd-4259-a649-80c8278cce9c">

## Solution

Add arialLable using i18n like in [this component](https://github.com/nodejs/nodejs.org/blob/004c16e039dbda73ab5048a607118ae27acca29f/components/Common/ThemeToggle/index.tsx#L21)

Useful ressource:

* [adding new translation keys](https://github.com/nodejs/nodejs.org/blob/main/TRANSLATION.md#adding-new-translation-keys)

**Comment by marwan-mohamed12** [comment]
I've Solved the issue see #6417 
Before
![image](https://github.com/nodejs/nodejs.org/assets/40841193/d6ac3e8d-0e4a-47f2-92b6-27158ca8f800)
After
![Screenshot 2024-03-03 172737](https://github.com/nodejs/nodejs.org/assets/40841193/a460c485-b64e-479a-8e58-6166267a169d)

**Comment by AugustinMauroy** [comment]
Fix on #6417
